lavamoat/browserify/beta/policy.json

```diff
@@ -10,11 +10,6 @@
         "regeneratorRuntime": "write"
       }
     },
-    "@download/blockies": {
-      "globals": {
-        "document.createElement": true
-      }
-    },
```

Comment:
I don’t know if this is generated or not, but I removed these from all the policy.json files. Let me know if there is another way to do it.

Reply:
The policy files are generated by running `yarn lavamoat:auto` but we can also use our bot to update them automatically so hold tight :)

Reply:
Cool :)

Reply:
Thanks :pray: